wandb/filesync/dir_watcher.py

```diff
@@ -314,8 +314,10 @@ def _get_file_event_handler(
         file_path: the file's actual path
         save_name: its path relative to the run directory (aka the watch directory)
         """
-        # Always return PolicyNow for any of our media files.
-        if save_name.startswith("media/"):
+        # Always return PolicyNow for any of our media files, except in importer mode
+        if (not os.getenv("WANDB_IMPORTER_MODE_ENABLED")) and save_name.startswith(
```

Comment:
once you use the importer in a process there is no way to go back to the "live" mode of media file uploads, which was the regression we needed to revert your previous pr and do a patch release.

Reply:
No, I think the the previous PR just commented this line out completely.  This instead will do an env check, so you can set/unset `WANDB_IMPORTER_MODE_ENABLED` to disable it.  Maybe we can unset the env var automatically in the importer

Reply:
How about something like this?
https://github.com/wandb/wandb/pull/6897/commits/83d985afd525224623dc794b00e5de0c9a293d18#diff-b04756399c59922236afaa7024b2a5260ad64d4ad0ebf305622257486239f93cR891-R937